src/middleware/jiraSymmetricJwtMiddleware.ts

```diff
@@ -0,0 +1,140 @@
+import { decodeSymmetric, getAlgorithm } from "atlassian-jwt";
```

Comment:
I think the name of the file is not following the convention, should be
`jira-symmetric-jwt-middleware.ts`